src/verilog.y

```diff
@@ -5394,66 +5394,71 @@ elseStmtBlock<nodep>:   // Part of concurrent_assertion_statement
 //UNSUP //                      //           id {variable_dimension} [ '=' property_actual_arg ]
 //UNSUP //                      // seq IEEE: [ yLOCAL [ sequence_lvar_port_direction ] ] sequence_formal_type
 //UNSUP //                      //           id {variable_dimension} [ '=' sequence_actual_arg ]
-//UNSUP         property_port_itemFront property_port_itemAssignment { }
-//UNSUP ;
+                property_port_itemFront property_port_itemAssignment { $$ = $2; }
+        ;
 
-//UNSUPproperty_port_itemFront: // IEEE: part of property_port_item/sequence_port_item
-//UNSUP         property_port_itemDirE property_formal_typeNoDt         { VARDTYPE($2); }
+property_port_itemFront: // IEEE: part of property_port_item/sequence_port_item
+//UNSUP            property_port_itemDirE property_formal_typeNoDt         { VARDTYPE($2); }
 //UNSUP //                      // data_type_or_implicit
-//UNSUP |       property_port_itemDirE data_type                { VARDTYPE($2); }
-//UNSUP |       property_port_itemDirE yVAR data_type           { VARDTYPE($3); }
-//UNSUP |       property_port_itemDirE yVAR implicit_typeE      { VARDTYPE($3); }
-//UNSUP |       property_port_itemDirE signingE rangeList       { VARDTYPE(SPACED($2,$3)); }
-//UNSUP |       property_port_itemDirE /*implicit*/             { /*VARDTYPE-same*/ }
-//UNSUP ;
+                   property_port_itemDirE data_type                { VARDTYPE($2); }
+//UNSUP    |       property_port_itemDirE yVAR data_type           { VARDTYPE($3); }
+//UNSUP    |       property_port_itemDirE yVAR implicit_typeE      { VARDTYPE($3); }
+//UNSUP    |       property_port_itemDirE signingE rangeList       { VARDTYPE(SPACED($2,$3)); }
+//UNSUP    |       property_port_itemDirE implicit_typeE           { VARDTYPE($2); }
+        ;
 
-//UNSUPproperty_port_itemAssignment<nodep>:  // IEEE: part of property_port_item/sequence_port_item/checker_port_direction
-//UNSUP         portSig variable_dimensionListE         { VARDONE($<fl>1, $1, $2, ""); PINNUMINC(); }
+property_port_itemAssignment<nodep>:  // IEEE: part of property_port_item/sequence_port_item/checker_port_direction
+                id variable_dimensionListE         { $$ = VARDONEA($<fl>1, *$1, $2, nullptr); }
 //UNSUP |       portSig variable_dimensionListE '=' property_actual_arg
 //UNSUP                 { VARDONE($<fl>1, $1, $2, $4); PINNUMINC(); }
-//UNSUP ;
+        ;
 
-//UNSUPproperty_port_itemDirE:
-//UNSUP         /* empty */                             { $$ = nullptr; }
-//UNSUP |       yLOCAL__ETC                             { }
-//UNSUP |       yLOCAL__ETC port_direction              { }
-//UNSUP ;
+property_port_itemDirE:
+/* empty */                             { GRAMMARP->m_pinAnsi = true; VARIO(INPUT); }
+//UNSUP   |       yLOCAL__ETC             { GRAMMARP->m_pinAnsi = true; VARIO(INPUT); }
+//UNSUP   |       yLOCAL__ETC yINPUT      { GRAMMARP->m_pinAnsi = true; VARIO(INPUT); }
```

Comment:
```suggestion
                /* empty */                             { GRAMMARP->m_pinAnsi = true; VARIO(INPUT); }
//UNSUP |        yLOCAL__ETC             { GRAMMARP->m_pinAnsi = true; VARIO(INPUT); }
//UNSUP |        yLOCAL__ETC yINPUT      { GRAMMARP->m_pinAnsi = true; VARIO(INPUT); }
```

Please match spacing of others; note should be "//UNSUP |" so the | is in same column as if UNSUP wasn't there.